src/app/components/authenticated-v2/subpages/armor-cluster-page/armor-cluster-page.component.html

```diff
@@ -1,150 +1,236 @@
-            <mat-button-toggle matTooltip="Titan" value="0">
-              <svg aria-hidden="true" class="svgClassIcon" focusable="false" viewBox="0 0 32 32"
-                   xmlns="http://www.w3.org/2000/svg"><path
-                d="m14.839 15.979-13.178-7.609v15.218zm2.322 0 13.178 7.609v-15.218zm5.485-12.175-6.589-3.804-13.178 7.609 13.178 7.609 13.179-7.609zm0 16.784-6.589-3.805-13.178 7.609 13.178 7.608 13.179-7.608-6.59-3.805z"
-                fill="currentColor"></path></svg>
-            </mat-button-toggle>
-            <mat-button-toggle matTooltip="Hunter" value="1">
-                <svg aria-hidden="true" class="svgClassIcon" focusable="false" viewBox="0 0 32 32"
-                     xmlns="http://www.w3.org/2000/svg"><path
-                  d="m9.055 10.446 6.945-.023-6.948 10.451 6.948-.024-7.412 11.15h-7.045l7.036-10.428h-7.036l7.032-10.422h-7.032l7.507-11.126 6.95-.024zm13.89 0-6.945-10.446 6.95.024 7.507 11.126h-7.032l7.032 10.422h-7.036l7.036 10.428h-7.045l-7.412-11.15 6.948.024-6.948-10.451z"
-                  fill="currentColor"></path></svg>
-              </mat-button-toggle>
-            <mat-button-toggle matTooltip="Warlock" value="2">
-                <svg aria-hidden="true" class="svgClassIcon" focusable="false" viewBox="0 0 32 32"
-                     xmlns="http://www.w3.org/2000/svg"><path
-                  d="m5.442 23.986 7.255-11.65-2.71-4.322-9.987 15.972zm5.986 0 4.28-6.849-2.717-4.333-6.992 11.182zm7.83-11.611 7.316 11.611h5.426l-10.015-15.972zm-7.26 11.611h8.004l-4.008-6.392zm6.991-11.182-2.703 4.324 4.302 6.858h5.413zm-5.707-.459 2.71-4.331 2.71 4.331-2.703 4.326z"
-                  fill="currentColor"></path></svg>
-              </mat-button-toggle>
-
-            <mat-button-toggle checked value="-1">All</mat-button-toggle>
-          </mat-button-toggle-group>
-        </span>
-      </div>
-    </mat-card-content>
-  </mat-card>
+    <a
+        class="returnbutton"
+        color="primary"
+        fxShow="false"
+        fxShow.lt-md="true"
+        mat-raised-button
+        routerLink="/">
+        Return to D2ArmorPicker
+    </a>
+    <mat-card>
+        <mat-card-title> About this page </mat-card-title>
+        <mat-card-content>
+            This page <i>can</i> help you to sort out your vault a bit. It takes all your armor
+            entries and tries to sort them into {{ clusterInformation.length }} clusters. It then
+            gives you a DIM search query for each cluster. You can use this to look at similar rolls
+            in your vault and to clean out some of them.
+            <br />
+            <br />
+            Please note that it will cluster all armor for all characters. Make sure to use
+            "is:titan", "is:warlock" or "is:hunter" in DIM.
+            <br />
+            <br />
+            Please also note that this feature is new and may have some inaccuracy for some items.
```

Comment:
not really a new feature anymore... but does still have the inaccuracy of course :D